apps/src/jigsaw/levels.js

```diff
@@ -131,8 +132,8 @@ module.exports = {
     },
     backgroundHSV: [184, 1.0, 0.733],
     ghost: {
-      x: 400,
-      y: 100,
+      x: 380,
```

Comment:
Thank you for your attention to detail here with multiples of the exact pixel-jump!